config/locales/nl.yml

```diff
@@ -0,0 +1,11 @@
+nl:
+  ibandit:
+    invalid_country_code: "'%{country_code}' is not a valid ISO 3166-1 IBAN country code"
```

Comment:
I'm pretty chilled about leaving these two missing (forgot to get them done, sorry!) - I'm not even convinced they should be translated at all as they're definitely not user-facing!